### ISSUE TITLE
systemctl show -p Property,Foo,Bar should print properties in the order requested on the command line

### ISSUE THREAD

**Issue by rpigott** [systemd/systemd#28205]
### Component

systemctl

### Is your feature request related to a problem? Please describe

`systemctl show -p Foo,Bar` currently prints properties in the order received from DBus. I think the utility of systemctl show for scripting could be greatly increased if we could control the order of the fields emitted from systemctl show, so that they could be more conveniently parsed by tools like awk, consumed by grep with context flags, used as line addresses with sed, etc. I figure most of the text based unix-y tools would appreciate this control.

### Describe the solution you'd like

`systemctl show -p Foo,Bar my.service` should emit

```
Foo=baz
Bar=quux
```

and never

```
Bar=quux
Foo=baz
```

### Describe alternatives you've considered

Sort the output manually, which might not be trivial when multiple unit arguments are supplied.

### The systemd version you checked that didn't have the feature you are asking for

253

**Comment by mbiebl** [comment]
appears to be a duplicate of #4907 which was closed with as not planned

**Comment by rpigott** [comment]
Fair enough, but #4907 also doesn't really seem to offer any justification for why the feature might be useful. I'd like to know if upstream is totally uninterested or would be willing to accept a PR implementing this behavior.

**Comment by devdekunle** [comment]
Hello @yuwata @DaanDeMeyer @poettering @bluca. Please I am interested in implementing this issue. I'd like to ask if upstream is interested in it being implemented.
